assets/js/editor-components/external-link-card/index.tsx

```diff
@@ -15,6 +16,7 @@ export interface ExternalLinkCardProps {
 	href: string;
 	title: string;
 	description?: string;
+	warning?: string;
```

Comment:
Can this value be `undefined`? If so, we should change it to `warning?: string | undefined;` based on https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tsconfig.base.json#L19

Reply:
@nielslange, this type is already defined as an [optional Parameters](https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters). The following type declarations are identical:

- `warning?: string | undefined;`
- `warning?: string;`